python/src/EnumerateFunctionImplementation_doc.i.in

```diff
@@ -4,19 +4,28 @@
 Notes
 -----
 *EnumerateFunction* represents a bijection from :math:`\Nset` to
-:math:`\Nset^{dim}`. This bijection is based on a particular procedure of
-enumerating the set of multi-indices. It begins from the multi-index
-:math:`(0, 0, \dots, 0)`.
+:math:`\Nset^{dim}`. This bijection is based on a particular enumeration rule
+of the set of multi-indices.
 
-For every integer :math:`i_p` in :math:`\Nset`, 
-we associate a multi-index :math:`\phi(i_p) = (j_{p1}, j_{p2},\dots, j_{pdim}) \in {\Nset}^{\textrm{dim}}`. 
+For every integer :math:`i \in \Nset`, we associate a multi-index
 
-Let :math:`i_p, i_q \in \Nset` be any pair of indices. 
-If :math:`|i_p-i_q|\leq 1` then
+.. math::
+
+    \boldsymbol{\alpha}(i) = (\alpha_1,\dots, \alpha_{n_X}) \in {\Nset}^{n_X}.
```

Comment:
To be consistent with the theory page, and assuming my previous suggestion is applied, this function should rather be called \vect{\tau}.